packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -66,6 +66,24 @@ $ContextualMenu-iconWidth: 14px;
   }
 }
 
+.ms-ContextualMenu-header {
+  @include focus-border();
+  @include ms-font-s;
+  font-weight: $ms-font-weight-semibold;
+  color: $ms-color-themePrimary;
+  background: none;
+  border: none;
+  height: $ContextualMenu-itemHeight;
+  line-height: $ContextualMenu-itemHeight;
+  cursor: default;
+  padding: 0px 6px;
+  user-select: none;
+  -moz-user-select: none;
+  -webkit-user-select: none;
```

Comment:
you dont' need the vendor prefixes. Postcss will take care of that for you. just use `user-select`.